### ISSUE TITLE
Adblock successful, Spotify fails to launch

### ISSUE THREAD

**Issue by HatshibaInu** [r1olo/spotify-adblock-macos#8]
Steps to reproduce: 
- Download Spotify, keep it open, and run the command in the folder as shown. 
- Quit and reopen Spotify. 
- Spotify no longer reopens - I'm guessing this has something to do with a new update. 

I'm guessing this is also a problem on your own machine - if you release an update, can you share it here too? :)

**Comment by ricardobalk** [comment]
Same, happened after update. Output of `/Applications/Spotify.app/Contents/MacOS/Spotify`:
```
dlsym /Applications/Spotify.app/Contents/MacOS/../Frameworks/Chromium Embedded Framework.framework/Chromium Embedded Framework: dlsym(0x2073d25a1, cef_shared_process_message_builder_create): symbol not found
Failed to load the CEF framework.
```

**Comment by ricardobalk** [comment]
I just proposed a fix: #9. If it gets accepted,

- Uninstall your current patch:
  ```sh
  rm '/Applications/Spotify.app/Contents/Frameworks/Chromium Embedded Framework.framework/Chromium Embedded Framework'
  ```
- Restore the original CEF
  ```sh
  mv '/Applications/Spotify.app/Contents/Frameworks/Chromium Embedded Framework.framework/Chromium Embedded Framework.orig' '/Applications/Spotify.app/Contents/Frameworks/Chromium Embedded Framework.framework/Chromium Embedded Framework'
  ```
- Patch again with `install.sh`.

That worked for me on Spotify version 1.1.93.896.g3ae3b4f3.
